melos.yaml

```diff
@@ -13,15 +13,22 @@ command:
     # branch: master
     # Additionally build a changelog at the root of the workspace.
     workspaceChangelog: true
+  bootstrap:
+    # It seems so that running "pub get" in parallel has some issues (like
+    # https://github.com/dart-lang/pub/issues/3404). Disabling this feature
```

Comment:
The mentioned issue already fixed, but no in Dart stable yet. We could try to enable `runPubGetInParallel` again in a few months.